addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -215,6 +215,23 @@ Wysiwyg.include({
         this.el.removeEventListener("click", this.__onPageClick, { capture: true });
         this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
+    openMediaDialog(params = {}) {
+        params.saveCallback = async (element) => {
+            const $element = $(element);
+            await this.snippetsMenu.callPostSnippetDrop($element);
```

Comment:
What's this? :thinking: No comment needed? :thinking:

Reply:
Added a comment. It's an alternative to `$element.click()` to make sure the options are built.